docs/community/tutorials-contest/2022/avalanche-theGraph-index/README.md

```diff
@@ -0,0 +1,381 @@
+## Indexing an Avalanche Local Subnet with The Graph
```

Comment:
Could you please decrease the the headings level? First one (the document title) should be a first heading level (single #), Introduction should be level 2 (##) and so on.

Reply:
This correction has been effected